template/js/netlify-cms/preview/home-preview.js

```diff
@@ -0,0 +1,164 @@
+const HomePreview = createClass({
+  render: function () {
+    return h('div', null,
+      // main
+      h('main', { role: 'admin', id: 'admin' },
+        // header
+        h('header', { className: 'header', id: 'header' },
+          // d-none d-sm-block
+          h('div', { className: 'd-none d-sm-block' },
+            h('a', { className: 'header__countdown' }, 'Enjoy our discounts of up to 40%')
```

Comment:
Por enquanto você está só mockando os elementos, esta frase _Enjoy our discounts of up to 40%_ por exemplo é só um padrão, mas editável...
Tornar o preview reativo às configurações feitas no CMS, certo?
Acho que fazer isto bem feito pro storefront inteiro só com render function (`h()`) vai ser quase impossível haha
Realmente precisamos importar e usar o React do jeito nutella aqui.

Reply:
É bem cansativo dessa forma mesmo, alem de ficar um pouco confuso. Mas eu farei usando o React.